authors/apps/pushnotifications/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db.models.signals import post_save
+from django.db import models
+from django.dispatch import receiver
+from django.conf import settings
+
+from pusher import Pusher
+
+from authors.apps.articles.models import Article, Like_Dislike
+from authors.apps.favorites. models import Favorites
+from authors.apps.comments.models import Comment
+from authors.apps.authentication.models import User
+
+pusher = Pusher(app_id=u'779166', key=u'931e65a8378bb81125c3',
```

Comment:
Is it possible for these to be added to the `.env`?

Reply:
Yes, @mwinel. You were very right. I incorporated your feedback in my new changes as I refactored and added some more functionality in the recent changes